api-docs/rst/dev-guide/api-operations/methods/put-update-launch-configuration-v1.0-tenantid-groups-groupid-launch.rst

```diff
@@ -169,6 +169,11 @@ This table shows the body parameters for the request:
 |                               |             |Rackspace Support after they configure your load   |
 |                               |             |balancer pool.                                     |
 +-------------------------------+-------------+---------------------------------------------------+
+|args.\ **draining_timeout**    |Integer      |Number of seconds deleting server's associated     |
+|                               |*(Optional)* |cloud load balancer node will be put in DRAINING   |
+|                               |             |before deleting the node followed by the server.   |
+|                               |             |Must be between 30 and 3600 inclusive.             |
```

Comment:
This sentence is a little confusing. I would rewrite it like this: "Specifies the number of seconds for which the cloud load balancer node associated with the server that is being deleted will be put in DRAINING before the node is actually being deleted followed by the server. Must be between 30 and 3600 inclusive.